pkgs/development/libraries/boost/generic.nix

```diff
@@ -171,7 +171,7 @@ stdenv.mkDerivation {
   postFixup = ''
     # Make boost header paths relative so that they are not runtime dependencies
     cd "$dev" && find include \( -name '*.hpp' -or -name '*.h' -or -name '*.ipp' \) \
-      -exec sed '1i#line 1 "{}"' -i '{}' \;
+      -exec sed 's/^\xef\xbb\xbf//;1i#line 1 "{}"' -i '{}' \;
```

Comment:
Nitpick just to be safe: can you please change this into `1s/^\xef\xbb\xbf//;` to make sure it only applies to line 1, as the BOM can only be in line 1.